src/entry_point_win.cpp

```diff
@@ -113,7 +114,29 @@ int WINAPI wWinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, LPWSTR
 		signal(SIGABRT, signal_abort_handler);
 	}
 
-	SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
+	// Workaround for old machines
+	HINSTANCE hUser32dll = LoadLibrary(L"User32.dll");
+	if(hUser32dll) {
+		auto pSetProcessDpiAwarenessContext = (decltype(&SetProcessDpiAwarenessContext))GetProcAddress(hUser32dll, "SetProcessDpiAwarenessContext");
+		if(pSetProcessDpiAwarenessContext != NULL) {
+			pSetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
+		} else {
+			// windows 8.1 (not present on windows 8 and only available on desktop apps)
+			HINSTANCE hShcoredll = LoadLibrary(L"Shcore.dll");
+			if(hShcoredll) {
+				auto pSetProcessDpiAwareness = (decltype(&SetProcessDpiAwareness))GetProcAddress(hShcoredll, "SetProcessDpiAwareness");
```

Comment:
We need someone to test whether these alternate paths actually work. The issue is that we draw over the whole window. Previously before PER_MONITOR_AWARE_V2 we might have had to call https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-enablenonclientdpiscaling on windows 10. So I have no idea whether this will actually work or not on window 8 as we expect